lib/bindings.js

```diff
@@ -1 +1,5 @@
-module.exports = require('bindings')('xmljs');
+var binary = require('node-pre-gyp');
+var path = require('path');
+var binding_path = binary.find(path.resolve(path.join(__dirname, '../package.json')));
```

Comment:
@bwknight877 I think you're right. Seeing as we'd have to define the binary path in two places using the bindings module, I'd rather just have it in package.json.

It would be nice if the bindings module or node-pre-gyp would automatically do this.